homeassistant/components/camera/webrtc.py

```diff
@@ -160,19 +320,73 @@ async def ws_get_client_config(
     )
 
 
-async def async_get_supported_providers(
-    hass: HomeAssistant, camera: Camera
-) -> list[CameraWebRTCProvider]:
-    """Return a list of supported providers for the camera."""
-    providers = hass.data.get(DATA_WEBRTC_PROVIDERS)
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "camera/webrtc/candidate",
+        vol.Required("entity_id"): cv.entity_id,
+        vol.Required("session_id"): str,
+        vol.Required("candidate"): str,
+    }
+)
+@websocket_api.async_response
+async def ws_candidate(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Handle WebRTC candidate websocket command."""
+    entity_id = msg["entity_id"]
+    camera = get_camera_from_entity_id(hass, entity_id)
+    if camera.frontend_stream_type != StreamType.WEB_RTC:
+        connection.send_error(
+            msg["id"],
+            "web_rtc_offer_failed",
```

Comment:
Isn't webrtc 1 word?

```suggestion
            "webrtc_offer_failed",
```